src/train/train_conv_net.py

```diff
@@ -1,6 +1,65 @@
 import tensorflow as tf
+from tensorflow.keras.preprocessing.image import ImageDataGenerator
 
 
 def train_conv_net():
     """ Train conv net on images """
-    pass
+    #HERE Get data and split it in diferent directories for the train generator
+    #HERE Split into train and test to verify that the code works
+
+    #HERE verify addressess and params
+    weights_address = 'backend/weights'
+    images_address_tr = 'train/images_tr'
+    images_address_tr = 'train/images_ts'
+    num_classes = 5
+    batch_size = 1000
```

Comment:
Batch size should be power of 2, see [this](https://datascience.stackexchange.com/questions/20179/what-is-the-advantage-of-keeping-batch-size-a-power-of-2)

Reply:
Also, slightly picky, but `num_classes` and `batch_size` should probably be final variable (declared at top all caps)